torrentfile/commands.py

```diff
@@ -417,6 +417,13 @@ def magnet(metafile: str, version: int = 0) -> str:
     trackers = "".join(announce_args)
 
     magnet += trackers if trackers != "&tr=" else ""
+    
+    if "url-list" in meta:
+        web-seed = [
```

Comment:
this is invalid syntax... python variables can only contain upper and lowercase letters number and an underscore,  the `-` character is not valid.